mobile/library/cc/engine_builder.cc

```diff
@@ -84,8 +84,8 @@ EngineBuilder& EngineBuilder::setOnEngineRunning(std::function<void()> closure)
   return *this;
 }
 
-EngineBuilder& EngineBuilder::addStatsSink(std::string name, std::string typed_config) {
-  stats_sinks_.push_back(std::make_pair(name, typed_config));
+EngineBuilder& EngineBuilder::addStatsSinks(const std::vector<std::string>& stats_sinks) {
```

Comment:
nit: This argument should be by value, not by const ref since it's being moved down below. This allows the caller to move the vector into this method and avoid a copy.